scripts/publish.js

```diff
@@ -72,9 +72,10 @@ const getPublishes = async ({ force }) => {
 }
 
 const main = async (opts) => {
-  const { isLocal, smokePublish, packDestination } = opts
-  const isPack = !!packDestination
-  const publishes = await getPublishes({ force: isPack })
+  const { test, otp, dryRun, smokePublish, packDestination } = opts
```

Comment:
destructing ALL flags here is really helpful to know what can be passed into this script

Reply:
Yes, do one or the other not both!